documentation/week3/src/App.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import './App.css';
+import Form from './components/Form';
+import List from './components/List';
+import data from './data/todos.json';
+import { loadFromLocalStorage, saveToLocalStorage } from './storage/localStoarge';
+export default class App extends Component {
+
+  state = loadFromLocalStorage() || {
+    todos: data,
+    id: 1,
+    discription: '',
+    deadline:''
+  }
+
+
+  handleDiscriptionChange = (event) => {
+    event.preventDefault();
+    this.setState({
+      discription: event.target.value
+    })
+  }
+
+  handleDeadlineChange = (event) => {
+    event.preventDefault();
```

Comment:
There's no need for `event.preventDefault()` here. We use it only when there's default behaviour for the HTML element that we are using, like `<form></form>`, because it has default behaviour.

Before Ajax, we were using <form method="GET" action="/submitAction.php"> tag with <input type="submit"> to make Backend calls directly without writing javascript code. That's why we're using `event.preventDefault()` .. to prevent this default behaviour.